### PR TITLE
Images drag drop

### PR DESCRIPTION
**Pull request by NaulaN**

### CODE REVIEW

File: src/css/global.css [hunk 1]
```diff
@@ -66,4 +66,12 @@ header {
 
 .account-menu > button {
     width: 100%;
+}
+
+#selected-item {
+    border: 1px solid blue;
+}
+
+#input-size {
```
Comment (almerion) [inline src/css/global.css:75]: no css in it ?

File: src/index.php [hunk 2]
```diff
@@ -41,6 +41,7 @@
 </body>
 <footer>
     <script src="./js/global.js" type="text/javascript"></script>
+    <script src="./js/movable-elements.js" type="text/javascript"></script>
     <script src="./js/draggable-elements.js" type="text/javascript"></script>
```
Comment (almerion) [inline src/index.php:45]: shouldn't these two scripts only be on the homepage ? 
they aren't needed on login/register page i think

File: src/css/global.css (hunk 1 above)
Comment (NaulaN) [reply src/css/global.css:75]: No needed if you look the JS script

**Comment by NaulaN** [comment]
@almerion You can check if my system for drag and drop an image is compatible with any other then Firefox browser ?

**Pull request merged**